Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -153,10 +153,10 @@ def wait_for_uninstallation_to_complete(client: demisto_client, retries: int = 3
                 raise Exception('Waiting time for packs to be uninstalled has passed, there are still installed '
                                 'packs. Aborting.')
             logging.info(f'The process of uninstalling all packs is not over! There are still {len(installed_packs)} '
-                         f'packs installed. Sleeping for 10 seconds.')
-            sleep(60)
+                         f'packs installed. Sleeping for 120 seconds.')
+            sleep(120)
```

Comment:
Put the value (120) in a variable, so you'd only need to change in one place.
```suggestion
            sleep_duration = 120
            logging.info(f'The process of uninstalling all packs is not over! There are still {len(installed_packs)} '
                                     f'packs installed. Sleeping for {sleep_duration} seconds.')
            sleep(sleep_duration)
```